engine/src/main/scala/cromwell/engine/workflow/lifecycle/WorkflowInitializationActor.scala

```diff
@@ -63,12 +63,11 @@ case class WorkflowInitializationActor(workflowId: WorkflowId, workflowDescripto
       val backendInitializationActors = Try {
         for {
           (backend, calls) <- workflowDescriptor.backendAssignments.groupBy(_._2).mapValues(_.keys.toSeq)
-          backendConfiguration = BackendConfiguration.backendConfigurationDescriptor(backend).get
           props <- CromwellBackends.shadowBackendLifecycleFactory(backend).map(factory =>
             factory.workflowInitializationActorProps(workflowDescriptor.backendDescriptor, calls)
           ).get
           actor = context.actorOf(props)
-        } yield (actor, backend)
```

Comment:
TL;DR of the previous comment-- the backend in the map values was never used, and I need to store engine actor refs in the map keys, now just a set of keys.